syntax/cpp.vim

```diff
@@ -50,6 +50,12 @@ if !exists("cpp_no_cpp14")
   syn match cppNumber		display "\<0b[01]\('\=[01]\+\)*\(u\=l\{0,2}\|ll\=u\)\>"
   syn match cppNumber		display "\<[1-9]\('\=\d\+\)*\(u\=l\{0,2}\|ll\=u\)\>"
   syn match cppNumber		display "\<0x\x\('\=\x\+\)*\(u\=l\{0,2}\|ll\=u\)\>"
+  syn match cppChronoLiteralsDecimal	display "\(\<0\|\<[1-9]\d*\)\(h\|min\|s\|ms\|us\|ns\)\>"
+  syn match cppChronoLiteralsFloat	display "\<\d\+\.\d*[eE]\=[\-+]\=\d\+\(h\|min\|s\|ms\|us\|ns\)\>"
+  syn match cppChronoLiteralsFloat	display "\.\d\+\([eE][\-+]\=\d\+\)\=\(h\|min\|s\|ms\|us\|ns\)\>"
+  syn match cppChronoLiteralsBinary	display "\<0b[01]\('\=[01]\+\)*\(h\|min\|s\|ms\|us\|ns\)\>"
+  syn match cppChronoLiteralsHexa	display "\<0x\x\('\=\x\+\)*\(h\|min\|s\|ms\|us\|ns\)\>"
+  syn match cppChronoLiteralsOctal	display "\<0\o\+\(h\|min\|s\|ms\|us\|ns\)\>" contains=cOctalZero
```

Comment:
If we determine to highlight them, how about adding them to existing number highlights as below?

```vim
syn match cppNumber		display "\<0x\x\('\=\x\+\)*\(u\=l\{0,2}\|ll\=u\|h\|min\|s\|ms\|us\|ns\)\>"
```